mtp_api/apps/core/filters.py

```diff
@@ -39,6 +39,23 @@ def filter(self, qs, value):
             return self.get_method(qs)(q).distinct()
         return self.get_method(qs)(q)
 
+    def label():
+        def fget(self):
+            if self._label is None and hasattr(self, 'parent'):
+                model = self.parent._meta.model
+                fields = []
+                for name in self.name:
+                    fields.append(django_filters.utils.label_for_filter(
+                        model, name, self.lookup_expr, self.exclude
+                    ))
+                self._label = ', '.join(fields)
+            return self._label
+
+        def fset(self, value):
+            self._label = value
+
+        return locals()
+
```

Comment:
I was mainly just copying what they had with the vague idea that it would make it easier to keep in line with upstream changes but that's probably not really a thing. Have changed to less gross version.